### ISSUE TITLE
Not work `go-task`

### ISSUE THREAD

**Issue by AlexanderZhirov** [getsolus/packages#864]
### Summary

![image](https://github.com/getsolus/packages/assets/37277056/d47b5973-32e5-47c1-a7f1-cf25f3e69c9b)

If you call the command that is specified in the instructions, then it does not work. And so it is with any data source.

### Steps to reproduce

Execute as in the instructions

### Expected result

Creating a `package.yml` file

### Actual result

`task: No Taskfile found at "/home/user/build/tree"`

### Environment

- [X] Is system up to date?

### Repo

Shannon (stable)

### Desktop Environment

MATE

### System details

OS: Solus x86_64
Kernel: 6.5.11-263.current

**Comment by malfisya** [comment]
You need this [file](https://github.com/getsolus/packages/blob/main/Taskfile.yml) at the root of your repo. Though my understanding of `go-task` that it meant to help with working in this [repository](https://github.com/getsolus/packages/) specifically. With `Taskfile.yml` and all that in `/common` needed.

**Comment by AlexanderZhirov** [comment]
I still can't understand the principle of creating a `package.yml` via 'go-task'. Why should I create any other files if I want to create just `package.yml` based on the source? Why is there no detailed description when working with `go-task` in the instructions?

**Comment by AlexanderZhirov** [comment]
In my opinion, the mechanism of a single package assembly was too complicated and scattered into different points of the instructions. Along with `Your First Package Update`, it would be possible to create `Your First Package Build`, where it is possible from the creation step `package.yml` before the package assembly step, explain the instructions in more detail. In the old manual, it was logically clear what needed to be done. In the current implementation, you've written it too hard. I had to go through all the points to understand that the package is being assembled instead of `make` now by calling `go-task`.

**Comment by EbonJaeger** [comment]
Just to make sure, have you gone through the steps in the Prepare for Packaging article, especially starting from [here](https://help.getsol.us/docs/packaging/prepare-for-packaging#fork-the-getsoluspackages-repository)? The root Taskfile is in the repo, so if you've cloned it, you would have it.

In August and September of this year, we moved from having many individual Git repositories for packages to having a single Git repo that contains every package, and from using Makefiles to go-task. More details on this can be found on [this page](https://help.getsol.us/docs/packaging/packaging-changes).

**Comment by ReillyBrogan** [comment]
Yes, `go-task` is a `make` replacement and serves the same role. It has a number of benefits to us, one of which is that it will traverse the directory tree until it finds a Taskfile.yml with the make/task definitions. This allows us to remove all the `Makefiles` that needed to be in every package folder. If you clone the package monorepo then there is already a `Taskfile.yml` in the root of it so go-task should work anywhere in it. If you are working in a separate repository of private packages you will probably want to copy the `Taskfile.yml` out of the Solus monorepo into your repository or perhaps make a symlink to it so it's kept updated with our changes (though I'm not sure if this will actually work or not since it has some relative imports).

**Comment by ReillyBrogan** [comment]
Also welcome back! Been a while since I've seen you around, hope you've been well

**Comment by AlexanderZhirov** [comment]
@ReillyBrogan Thank you, everything is great! 😊 Well, now I have to master a new toolkit 😁
